src/App.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
+
```

Comment:
remove unnecessary line
```suggestion
```

Reply:
its line to divide import from third part library and my code